src/features/unified-share-modal/UnifiedShareModal.js

```diff
@@ -164,12 +165,12 @@ class UnifiedShareModal extends React.Component<Props, State> {
 
         this.state = {
             emailSharedLinkContacts: [],
-            inviteCollabsContacts: props.initiallySelectedContacts || [],
+            inviteCollabsContacts: props.initiallySelectedContacts,
```

Comment:
Also, shouldn't there be a unit test, to test that initial state is as expected?
If there is none, should add one. You can disable lifecycle methods so that the constructor is the only one executed besides render.